server/db/models/Product.js

```diff
@@ -1,12 +1,54 @@
-const { STRING } = require('sequelize')
+const { STRING,TEXT,INTEGER,DECIMAL } = require('sequelize')
 const db = require('../db')
 
+const Product = db.define('product', {
+    title: {
+        type: STRING,
+        allowNull: false,
+        validate: {
+            notEmpty: true,
+        },
+    },
+    brand: {
+        type: STRING,
+        allowNull: false,
+    },
+    category: {
+        type: STRING,
+        allowNull:false
+    },
+    description: {
+        type: TEXT,
+        allowNull: false,
+    },
+    price: {
+        type: DECIMAL(10, 2),
+        validate: {
+            isDecimal: true,
+        },
+    },
+    inventory: { type: INTEGER,
+               defaultValue:0},
+    country: { type: STRING,
+             allowNull: false,
+        validate: {
+            notEmpty: true,
+        }},
+    imageUrl: {
+        type: STRING,
+        validate: {
+            isUrl: true,
+        }
```

Comment:
You're miss a comma on line 41 after the }

Reply:
Thanks!